### ISSUE TITLE
Post variables so that they are available for use in html content creator

### ISSUE THREAD

**Issue by etameez** [upfrontsystems/izinto-angular#218]
Currently, the variables in the variables tab are not accessible in the html content window (I may be accessing them incorrectly?)

I think it just needs to be posted to the iframe as done for the dateSelection in app/dashboard/dashboard.component.ts 
Lines 87-95.
![image](https://user-images.githubusercontent.com/37751346/97850451-bb05fc00-1cfc-11eb-9380-e3ce720f2897.png)

**Comment by nvernooy** [comment]
Addressed in #13

**Comment by nvernooy** [comment]
Variables can be requested by calling
```
window.top.postMessage({'variables': true}, '*');
```
And by receiving response 
```
if (event.data.type === 'variables') {
    variables = event.data.message;
}
```
